### ISSUE TITLE
BUG: Gitpod terminal not showing latest version of pymc

### ISSUE THREAD

**Issue by reshamas** [pymc-devs/pymc#6600]
### Describe the issue:

I was going through the Gitpod process again. I started from the beginning (forked my repo), etc. And this time it is showing an older version of pymc.

(When I did it a few days ago, it was showing version 5.1.0).  

### Reproduceable code example:

```python
(base) gitpod@reshamas-pymc-d0ep89s0e45:/workspace/pymc$ git remote -v
origin  https://github.com/reshamas/pymc.git (fetch)
origin  https://github.com/reshamas/pymc.git (push)
upstream        https://github.com/pymc-devs/pymc.git (fetch)
upstream        https://github.com/pymc-devs/pymc.git (push)
(base) gitpod@reshamas-pymc-d0ep89s0e45:/workspace/pymc$ pip list | grep pymc
pymc                          4.4.0+352.g56037929 /workspace/pymc
pymc-sphinx-theme             0.1
(base) gitpod@reshamas-pymc-d0ep89s0e45:/workspace/pymc$ python3 --version
Python 3.11.0
(base) gitpod@reshamas-pymc-d0ep89s0e45:/workspace/pymc$
```

### PyMC version information:

PyMC v5

**Comment by reshamas** [comment]
cc:  @maresb

**Comment by reshamas** [comment]
@maresb 
I spun up Gitpod and good news is it is now showing version `5.1.1`: 
```console
(base) gitpod@reshamas-pymc-syxfrf90fp0:/workspace/pymc$ pip list | grep pymc
pymc                          5.1.1+37.g83cd926e /workspace/pymc
pymc-sphinx-theme             0.1
(base) gitpod@reshamas-pymc-syxfrf90fp0:/workspace/pymc$ 
```

It's not 4.4.0, which is good. However it is not showing 5.1.2, which is the current one.

**Comment by maresb** [comment]
Ya, I think this is just versioneer weirdness, and nothing to be worried about. I will try to fix it when I can find time

**Comment by reshamas** [comment]
2023-03-30 email from Gitpod

Hey there 👋

On April 3rd, Gitpod will upgrade the default workspace images to utilize the latest Long Term Support (LTS) version of Ubuntu. There are potential breaking changes depending on the current workspace image you are using. 

If you do experience any problems with your workspace, kindly submit an issue in the [workspace-images](http://email.gitpod.io/e/c/eyJlbWFpbF9pZCI6ImRnVGl2Z1lEQU9XM0YtUzNGd0dITWxCazlCLXByRy1MWEVzMjFiTT0iLCJocmVmIjoiaHR0cHM6Ly9naXRodWIuY29tL2dpdHBvZC1pby93b3Jrc3BhY2UtaW1hZ2VzIiwiaW50ZXJuYWwiOiJlMmJlMDYwY2E0YmIwMWU1YjcxNyIsImxpbmtfaWQiOjUyNzJ9/ff15f3aba7ee07b057d34424f3b78dac38654d472318c898c3803e86bbfc67f5) repository.

In the meantime, we recommend following the steps [outlined here](http://email.gitpod.io/e/c/eyJlbWFpbF9pZCI6ImRnVGl2Z1lEQU9XM0YtUzNGd0dITWxCazlCLXByRy1MWEVzMjFiTT0iLCJocmVmIjoiaHR0cHM6Ly93d3cuZ2l0cG9kLmlvL2NoYW5nZWxvZy93b3JrcGFjZS1pbWFnZXMtb3MtdXBkYXRlLWFuZC1icmVha2luZy1jaGFuZ2VzIiwiaW50ZXJuYWwiOiJlMmJlMDYwY2E0YmIwMWU1YjcxNyIsImxpbmtfaWQiOjUyNzN9/bb98d76fde653053e1d17d6898a94ce98cb45e2f40a3cd2645b72a971ad23539). 

Happy coding,

The Gitpod Team

**Comment by maresb** [comment]
We are using a customized dev container, so this change will not affect our Gitpod setup.

**Comment by ricardoV94** [comment]
Is this still a problem @reshamas @maresb ?

**Comment by maresb** [comment]
We may want to update the docs at https://www.pymc.io/projects/docs/en/latest/contributing/using_gitpod.html

We might want to add some instructions for updating the version; something like:

```bash
cd /workspace/pymc
git checkout main
git pull upstream main
pip install -e .
python -c "import pymc; print(pymc.__version__)"
```

**Comment by reshamas** [comment]
@maresb 
Is this expected, the part `+303.g6f8f9eef`?
```
5.1.1+303.g6f8f9eef
```

---

```bash
(base) gitpod@reshamas-pymc-syxfrf90fp0:/workspace/pymc$ pip install -e .
Obtaining file:///workspace/pymc
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Preparing editable metadata (pyproject.toml) ... done
Requirement already satisfied: arviz>=0.13.0 in /opt/conda/lib/python3.11/site-packages (from pymc==5.1.1+303.g6f8f9eef) (0.15.1)
...
  Building editable for pymc (pyproject.toml) ... done
  Created wheel for pymc: filename=pymc-5.1.1+303.g6f8f9eef-0.editable-py3-none-any.whl size=11527 sha256=6211b7149b3ab09813b2badb3010f54d2d4ab014f75054d73f204ac5ea82ed82
  Stored in directory: /tmp/pip-ephem-wheel-cache-wmkfx8pd/wheels/73/bf/14/341b7fa040e9af1991e12077c13913921be3069fe3bdf78752
Successfully built pymc
Installing collected packages: pytensor, pymc
  Attempting uninstall: pytensor
    Found existing installation: pytensor 2.10.1
    Uninstalling pytensor-2.10.1:
      Successfully uninstalled pytensor-2.10.1
Successfully installed pymc-5.1.1+303.g6f8f9eef pytensor-2.18.6

(base) gitpod@reshamas-pymc-syxfrf90fp0:/workspace/pymc$ python -c "import pymc; print(pymc.__version__)"
5.1.1+303.g6f8f9eef

(base) gitpod@reshamas-pymc-syxfrf90fp0:/workspace/pymc$ pip list | grep pymc
pymc                          5.1.1+303.g6f8f9eef /workspace/pymc
pymc-sphinx-theme             0.1

(base) gitpod@reshamas-pymc-syxfrf90fp0:/workspace/pymc$ 
```

**Comment by maresb** [comment]
Yes, it indicates 303 commits after the 5.1.1 tag with Git sha 6f8f9eef.

**Comment by maresb** [comment]
@reshamas, thanks so much for tackling this in #7106. There I suggested an update to the `git pull` command:

```bash
git pull upstream main --tags
```

I'd expect running this command followed by `pip install -e .` should hopefully have the version report relative to the latest release tag instead of the old `5.1.1` you were seeing above.

**Comment by reshamas** [comment]
@maresb @ricardoV94 
Should we close this issue?

**Comment by maresb** [comment]
Yes, closed by #7106. Thanks @reshamas!
